TriangleLib/TriangleLib/Triangle.cs

```diff
@@ -4,5 +4,150 @@ namespace TriangleLib
 {
     public class Triangle
     {
+		int Aside;
+		int Bside;
+		int Cside;
```

Comment:
doesn't follow naming convention, should be e.g. `_aSide`

Reply:
You right.